### DISCUSSION TITLE
敏實科大攜手數控大廠 培育人工智慧專業人才

### DISCUSSION THREAD

**Discussion by ghost** [1101-GS4719-C/Python-Community-GS4719-C#496] [General]
"敏實科技大學與國內知名控制器廠商寶元數控(LNC)公司策略聯盟成立「AI人工智慧、機器人培訓中心」，提供LNC EtherCat系列數控系統、機器人與周邊系統設備以及AI人工智慧系統等，首先建置「人工智慧、機器人、CNC控制器應用體驗區」作為敏實科大人工智慧學院教學研發之用。敏實科大校長曾信超表示，未來計劃合作訓練新一代智慧製造人才，並辦理全國應用競賽活動，推動培育國內工業4.0及智能科技人才。"

說真的蠻希望中央有類似的技術培訓中心XD不過中央本身也不是科技大學就是了，但蠻希望能有更多的教技職中心之類的，讓中央能有相比其他校不同的優勢
文中的智慧生產系統或是加工機，再製造的自動化上都能有一定的期待性，雖然這也另類的體現出相關領域的職業該何去何從

https://tw.news.yahoo.com/%E6%95%8F%E5%AF%A6%E7%A7%91%E5%A4%A7%E6%94%9C%E6%89%8B%E6%95%B8%E6%8E%A7%E5%A4%A7%E5%BB%A0-%E5%9F%B9%E8%82%B2%E4%BA%BA%E5%B7%A5%E6%99%BA%E6%85%A7%E5%B0%88%E6%A5%AD%E4%BA%BA%E6%89%8D-054950284.html?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuYmluZy5jb20v&guce_referrer_sig=AQAAAKAhgyAQj5qOVi4CqlTyP07iRs0lhoaecK-A-hwyfoIrjvtOP0h_fX-n-ft8tB4h6NYcdBReO7lLAgIbpF0ny6eCKlGEo_hOthvgRRs8KljW4Y3B46uG5e19-HevrxoEttwBh-5980tvOMAjE5vay25Z606VyA3QWZic4JcsXJZ3

**Comment by ThomasMeng0515** [comment]
第一次聽到敏實科技大學，我太孤陋寡聞了... 但普通大學沒有這些技職中心感覺很正常

**Comment by sean1325418** [comment]
大學就是能學越多東西越好，最好學的能跟未來職場有關。

**Comment by xkllkx** [comment]
我是覺得就算沒有學校的資源，網路上也有許多其他資源能夠自學。

**Comment by chupochun** [comment]
謝謝分享，讓我了解在中央的我們還有很多比不上別人的地方

**Comment by Yuhsin000** [comment]
謝謝分享，多認識了一所學校，產學合作才能更近實際產業，這是科技大學的優勢之一，希望我們也能向他們學習，增加自己的優勢。

**Comment by PoooYen** [comment]
目前有些科技大學在硬、軟體方面的著墨的確比普通大學還要來的深厚
